star-ros2/src/star_gateway_bridge/include/star_gateway_bridge/message_converter.hpp

```
@@ -58,11 +57,10 @@ class MessageConverter {
    * @param sequence Optional sequence number for command ordering
    * @return true if conversion successful, false if input validation failed
    */
-  static bool twist_to_velocity_command(
-    const geometry_msgs::msg::Twist & twist,
-    star::v1::VelocityCommand & command,
-    double wheel_base = 0.150,
-    uint32_t sequence = 0);
+  static bool twist_to_velocity_command(const geometry_msgs::msg::Twist &twist,
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)